jmx-metrics/src/integrationTest/java/io/opentelemetry/contrib/jmxmetrics/target_systems/KafkaIntegrationTest.java

```diff
@@ -28,6 +30,29 @@
 import org.testcontainers.utility.MountableFile;
 
 abstract class KafkaIntegrationTest extends AbstractIntegrationTest {
+  Startable createTopics =
```

Comment:
Any reason for moving this? Previously the containers were listed in order of execution which was a bit nicer

Reply:
Nope. I had moved it initially when I was trying to leverage it for another container but ended up removing that and didn't move it back. Will resolve.